app/jobs/nsm/schedule_delete_reviewed_docs.rb

```diff
@@ -0,0 +1,24 @@
+module Nsm
+  class ScheduleDeleteReviewedDocs < ApplicationJob
+    sidekiq_options retry: 5
+
+    def perform
+      return if filtered_claims.empty?
+
+      filtered_claims.each do |claim|
+        Nsm::DeleteReviewedClaimDocs.perform_later(claim.id)
+      end
+    end
+
+    def filtered_claims
+      state = %w[granted part_grant rejected expired]
+
```

Comment:
maybe comments here would be good to express the sentiment of what you want this query to be?

Reply:
this is a nit so resolve if you want